src/main/java/com/gitrekt/resort/controller/FeedbackReportItemScreenController.java

```diff
@@ -0,0 +1,20 @@
+package com.gitrekt.resort.controller;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.fxml.Initializable;
+
+/**
+ * Handles the functionality of FeedbackReportItemScreen.fxml
+ */
+public class FeedbackReportItemScreenController implements Initializable {
```

Comment:
A better name would have been FeedbackScreenItemController; in addition to being shorter, a list item is not a screen. In many ways it does behave like one, but that does not make it a screen. Sorry if I confused you earlier with that analogy.